motr/st/utils/helper.c

```diff
@@ -250,6 +254,12 @@ static int create_object(struct m0_entity *entity)
 			       M0_BITS(M0_OS_FAILED,
 				       M0_OS_STABLE),
 				       M0_TIME_NEVER);
+	oc = M0_AMB(oc, ops[0], oc_op);
+	oo = M0_AMB(oo, oc, oo_oc);
+	obj = m0__obj_entity(oo->oo_oc.oc_op.op_entity);
+	M0_LOG(M0_DEBUG, "post create object, obj->ob_attr.oa_pver :"FID_F,
+	       FID_P(&obj->ob_attr.oa_pver));
```

Comment:
Move these lines after checking rc.
```
        if (rc == 0)
		rc = ops[0]->op_rc;
	oc = M0_AMB(oc, ops[0], oc_op);
	oo = M0_AMB(oo, oc, oo_oc);
	obj = m0__obj_entity(oo->oo_oc.oc_op.op_entity);
	M0_LOG(M0_DEBUG, "post create object, obj->ob_attr.oa_pver :"FID_F" rc=%d",
	                                      FID_P(&obj->ob_attr.oa_pver), rc);
```
Checking result must be immediately after the function all. Otherwise, it is error-prone.